src/transform/CollapseTableTransform.js

```diff
@@ -177,24 +177,24 @@ const newCaption = (title, headerText) => {
  * @param {?string} infoboxTitle
  * @param {?string} otherTitle
  * @param {?string} footerTitle
- * @param {?FooterDivClickCallback} footerDivClickCallback
+ * @param {?FooterCallback} FooterCallback
  * @return {void}
  */
-const collapseTables = (window, content, pageTitle, isMainPage, infoboxTitle, otherTitle,
-  footerTitle, footerDivClickCallback) => {
+const transform = (window, content, pageTitle, isMainPage, infoboxTitle, otherTitle,
```

Comment:
This does more than just collapse the tables though. It's a transformation that enables tables that can collapse or expand and collapse just happens to be the default state.